cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -163,6 +163,7 @@ private String getEncryptedPassword(String variable) {
 
     ProcessContext pc = this.context.newProcess().executable("mvn");
     pc.addArgs("--encrypt-password", input);
+    pc.addArg("-Dsettings.security=" + this.context.getMavenRepository().getParent().resolve(SETTINGS_SECURITY_FILE).toString().replace("\\", "\\\\"));
```

Comment:
@KianRolf this will fix the password encryption but not the decryption that later needs to happen in order to make use of the password.